### PR TITLE
Tailwind setup

### PR DESCRIPTION
**Pull request by minisodan**
## what changed
- Ended up settling with Tailwind, added starter code.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /**
  * Entry function from main.tsx which will be the main entry point for the differing Components
  */
-const App = () => {};
+const App = () => {
```
Comment (BowlBird) [inline src/App.tsx]: remove before merging to main

**Review by BowlBird — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (minisodan) [reply src/App.tsx]: sounds good, just a place holder to make sure its working

File: src-tauri/tauri.conf.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
       {
         "title": "notepad.me",
         "width": 800,
-        "height": 600
+        "height": 600,
+        "decorations": false
```
Comment (BowlBird) [inline src-tauri/tauri.conf.json]: I think this came over from the decorations branch on accident

**Review by BowlBird — CHANGES_REQUESTED** [review]

File: src-tauri/tauri.conf.json [hunk 3]
```diff
@@ -13,9 +13,7 @@
     "windows": [
       {
         "title": "notepad.me",
-        "width": 800,
-        "height": 600,
```
Comment (BowlBird) [inline src-tauri/tauri.conf.json]: height shouldn't be deleted from here

**Review by BowlBird — APPROVED** [review]

**Pull request merged**